lib/middleware/csrf.js

```diff
@@ -0,0 +1,34 @@
+const Tokens = require('csrf');
+
+const checkSession = (req, res, next) => {
+  if (req.session && req.session.form && req.session.form[req.model.id]) {
+    return next();
+  }
+  const err = new Error('Form session not set');
+  err.code = 500;
+  return next(err);
```

Comment:
You _probably_ want to set `status` rather than `code` here, but what you actually probably want is just `next(new Error('Form session not set'));` because the status defaults to 500 unless otherwise specified.